src/babashka/nrepl/impl/server.clj

```diff
@@ -207,14 +247,20 @@
         :complete (do
                     (complete ctx os msg opts)
                     (recur ctx is os id opts))
+        :lookup (do
```

Comment:
maybe:

```
(:lookup :info) (do ...)
```
?

Reply:
nice, I didn't know this form was supported in `case` :+1: